ee/dataflatten/examples/flatten/main.go

```diff
@@ -40,13 +40,14 @@ func main() {
 		checkError(fmt.Errorf("parsing flags: %w", err))
 	}
 
-	logger := logutil.NewCLILogger(*flDebug)
-
 	opts := []dataflatten.FlattenOpts{
-		dataflatten.WithLogger(logger),
+		dataflatten.WithSlogger(multislogger.New().Logger),
```

Comment:
This is example code. It feels slightly weirf to pass `multislogger`. Can we pass a simple slogger instead?

Reply:
This is what we tend to copy-paste everywhere we need an unused slogger -- e.g. it's all over our unit tests. I'm not sure that this is wrong.

Reply:
:shrug: